src/duke/Duke.java

```diff
@@ -0,0 +1,36 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Duke {
+    private Storage s;
+    private TaskList tasks;
+    private Ui ui;
+
+    public Duke(String filePath) throws Exception {
+
+        s = new Storage(filePath);
+        s.create();
+        try {
+            tasks = new TaskList(s.toArrayList());
+            ui = new Ui(tasks, s);
+        } catch (Exception e) {
+            System.out.println(e);
+        }
+    }
+
+    /**
+     * Runs the whole program.
+     * @throws Exception
```

Comment:
What is this exception? When would it be thrown? Perhaps it is better to specify in the JavaDoc. Same for other JavaDoc comments for @throws tag.